### PR TITLE
Document Upper/LowerTriangular

### PR DESCRIPTION
**Pull request by fredrikekre**
Document `UpperTriangular`/`LowerTriangular` and clarify that `Symmetric`/`Hermitian` are views of the underlying matrix as well.

### CODE REVIEW

**Comment by fredrikekre** [comment]
Failure is https://github.com/JuliaLang/julia/pull/22472#issuecomment-310820067

**Comment by tkelman** [comment]
We didn't have this yet? LGTM

File: base/linalg/symmetric.jl [hunk 1]
```diff
@@ -49,7 +50,8 @@ end
 """
     Hermitian(A, uplo=:U)
 
-Construct a `Hermitian` matrix from the upper (if `uplo = :U`) or lower (if `uplo = :L`) triangle of `A`.
+Construct a `Hermitian` view from the upper (if `uplo = :U`) or lower (if `uplo = :L`)
```
Comment (KristofferC) [inline base/linalg/symmetric.jl]: What is best, "from the upper" or "of the upper"?

Comment (fredrikekre) [inline base/linalg/symmetric.jl]: "of the upper" maybe?

**Review by Sacha0 — APPROVED** [review]
Fantastic! :)

**Review by KristofferC — APPROVED** [review]

**Comment by fredrikekre** [comment]
Thanks!

**Pull request merged**